docs/developing.md

```diff
@@ -47,12 +47,13 @@ This repository uses [Nuke](https://nuke.build/) for build automation.
 
 Support plugins are available for:
 
-- JetBrains ReSharper        https://nuke.build/resharper
-- JetBrains Rider            https://nuke.build/rider
-- Microsoft VisualStudio     https://nuke.build/visualstudio
-- Microsoft VSCode           https://nuke.build/vscode
+- JetBrains ReSharper        <https://nuke.build/resharper>
+- JetBrains Rider            <https://nuke.build/rider>
+- Microsoft VisualStudio     <https://nuke.build/visualstudio>
+- Microsoft VSCode           <https://nuke.build/vscode>
 
-Restore dotnet tools to prepare build tools for solution. This installs the dotnet `nuke` tool locally.
+Restore dotnet tools to prepare build tools for solution.
+This installsthe dotnet `nuke` tool locally.
```

Comment:
Space is missing after installs

Reply:
fixed f4bbf0d9bbd8ec7d1413f3215080ec80417b3cce